lightwood/mixers/nn.py

```diff
@@ -489,29 +440,12 @@ def _predict(self, when_data_source, include_extra_data=False):
             if when_data_source.out_types[k] in (COLUMN_DATA_TYPES.NUMERIC):
                 predictions[output_column] = {'predictions': [x[0] for x in decoded_predictions]}
 
-                if include_extra_data:
-                    predictions[output_column]['every_confidence_range'] = [x[1:] for x in decoded_predictions]
-
             else:
                 predictions[output_column] = {'predictions': decoded_predictions}
 
             if awareness_arr is not None:
                 predictions[output_column]['selfaware_confidences'] = [1/abs(x[k]) if x[k] != 0 else 1/0.000001 for x in awareness_arr]
 
-            if when_data_source.out_types[k] in (COLUMN_DATA_TYPES.NUMERIC):
-                predictions[output_column]['confidence_range'] = []
-                predictions[output_column]['quantile_confidences'] = []
-
-                for i, pred in enumerate(decoded_predictions):
```

Comment:
This bit (`selfaware_confidences` key) is still used by the nonconformist normalizer

Reply:
The `selfaware_confidences` confidence is not removed, only the `sc` which is a reference to it used to select the quantile

Reply:
Ah, I missed that, you're right